lib/tasks/turbo_tasks.rake

```diff
@@ -1,6 +1,22 @@
 namespace :turbo do
-  desc "Install Turbo into the app"
+  desc "Install Turbo into the app. If webpacker is configured delegates to webpacker, else uses asset pipline"
   task :install do
-    system "#{RbConfig.ruby} ./bin/rails app:template LOCATION=#{File.expand_path("../install/turbo.rb", __dir__)}"
+    if File.exist? 'bin/webpack'
```

Comment:
Maybe there is a better way to check for `webpacker` ?

Reply:
Perhaps ```!!defined?(Webpacker)```